ferretdb_packaging/development.Dockerfile

```diff
@@ -0,0 +1,60 @@
+# syntax=docker/dockerfile:1
```

Comment:
Do we need language/local related things?  https://github.com/FerretDB/FerretDB/blob/main/build/postgres-documentdb/development.Dockerfile
```
ENV LANG=en_US.UTF-8
ENV LANGUAGE=en_US
ENV LC_ALL=en_US.UTF-8
ENV LC_CTYPE=en_US.UTF-8
```

Reply:
I think we set them to avoid build failures at some step. They might already be set by a base image now. Let's try removing them.

Reply:
Build and run both don't fail without them now :+1:
Base container sets `ENV LANG=en_US.utf8`, and it seems to be sufficient.